lib/fly.js

```diff
@@ -61,6 +57,14 @@ function Fly(options) {
 		}, {})
 	})
 
+	if (!file) {
```

Comment:
If we declare it higher tests will fail because we did not assign default params like _.filters and so on.